extension/e2e-tests/onboarding.test.ts

```diff
@@ -62,7 +62,11 @@ test("Create new wallet", async ({ page }) => {
     await page.getByLabel(words[i]).check({ force: true });
   }
   await page.getByTestId("display-mnemonic-phrase-confirm-btn").click();
-  await expect(page.getByText("You’re all set!")).toBeVisible();
+
+  // Wait for navigation to success page (create wallet uses different route than import)
```

Comment:
there's a lot of seemingly unrelated changes to tests in this PR. Was this the only way you could get your tests to run? It'd be helpful from a reviewer perspective to understand why we need all these changes